src/main/default/layouts/Trigger_Handler__mdt-Trigger Handler Layout.layout-meta.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Layout xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
probably worth structuring the layout a bit

Reply:
Did a bit more layout prettifying and help text. Removed object name field since we should just use the developername of the metadata for objects.